net-lib/net/dns.rkt

```diff
@@ -14,6 +16,28 @@
            (->* ((or/c ip-address? ip-address-string?) string?)
                 (#:ipv6? any/c)
                 ip-address-string?)]
+          [dns-get-srv
+           (->* ((or/c ip-address? ip-address-string?)
+                 string?
+                 string?)
+                (string?)
+                (listof (struct/c srv-rr
+                                  (integer-in 0 65535)
+                                  (integer-in 0 65535)
+                                  (integer-in 0 65535)
+                                  string?)))]
+          [dns-lookup
+           (-> (or/c ip-address? ip-address-string?)
+               string?
+               #:rr-type symbol?
+               #:rr-parser (-> (listof (list/c bytes? ;; name
```

Comment:
If we're going to expose a general DNS record lookup function, would it make sense to produce some record structs here? (rather than lists-as-tuples)